webapp/backend/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+    "semi": true,
+    "singleQuote": false
+}
```

Comment:
add a newline at the end of the file to remove the mark here. if you hover over it you'll see what the issue is as well

more details on it here https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log